rest-api/src/main/java/gov/cms/qpp/conversion/api/services/StorageServiceImpl.java

```diff
@@ -107,19 +112,20 @@ public InputStream getFileByLocationId(String fileLocationId) {
 	 * @return file used for cpc+ validation.
 	 */
 	@Override
-	public InputStream getCpcPlusValidationFile() {
-		String bucketName = environment.getProperty(Constants.CPC_PLUS_BUCKET_NAME_VARIABLE);
-		String key = environment.getProperty(Constants.CPC_PLUS_FILENAME_VARIABLE);
-		if (StringUtils.isEmpty(bucketName) || StringUtils.isEmpty(key)) {
-			API_LOG.warn("No CPC+ bucket name and/or CPC+ key specified");
+	public byte[] getCpcPlusValidationFile() {
+		if (StringUtils.isEmpty(Constants.FMS_TOKEN)) {
```

Comment:
Will the FMS token be stored in S3 as an env variable? Because that constant will always be empty.